openmc/deplete/results_list.py

```diff
@@ -84,13 +84,14 @@ def get_atoms(self, mat, nuc, nuc_units="atoms", time_units="s"):
         cv.check_value("nuc_units", nuc_units,
                     {"atoms", "atom/b-cm", "atom/cm3"})
 
+        mat_id = str(mat.id)
```

Comment:
Based on the above comment
```suggestion
        if isinstance(mat, openmc.Material):
            mat = mat.id
```

Reply:
Done as suggested